packages/wells/src/client/api/surveysApi.ts

```diff
@@ -25,6 +29,17 @@ export class SurveysAPI extends ConfigureAPI {
       });
   };
 
+  public getTrajectories = async (wellboreIds: number[]): Promise<Survey[]> => {
+    const path: string = this.getPath(`/wellbores/trajectories`);
+
+    return await this.client
+      .asyncPost<Survey[]>(path, { data: { items: wellboreIds } })
+      .then(response => this.addLazyMethodsToAll(response.data))
+      .catch(err => {
+        throw new HttpError(err.status, err.errorMessage, {});
```

Comment:
this should be `throw new HttpError(err.status, err.data.error.message, {});`. Could you change this on line 98 in `wellsApi.ts` as well?